internal/handlers/pg/pgdb/pool.go

```diff
@@ -343,10 +347,38 @@ func (pgPool *Pool) DropDatabase(ctx context.Context, db string) error {
 	}
 }
 
+// validateCollectionName validates collection name.
+// Collection name must not contain non-latin letters, spaces, dots, dollars, dashes,
+// name collection name length must be less than 119 characters.
+func (pgPool *Pool) validateCollectionName(collection string) error {
+	if len(collection) == 0 {
+		return ErrInvalidTableName
+	}
+
+	if len(collection) > 119 {
+		return ErrInvalidTableName
+	}
```

Comment:
I think those are redundant checks, as they are included in regex. Therefore they can be removed.